src/AutoRest.CSharp/Common/AutoRest/Plugins/NewProjectScaffolding.cs

```diff
@@ -138,6 +138,28 @@ private string GetChangeLog()
 
         private string GetReadme()
         {
+            const string multipleApiVersionContent = @"
+
+### Service API versions
+
+The client library is designed to communicate with multiple service API versions. By default, the client library targets the latest service API version.
```

Comment:
I feel `designed` set unrealistic expectations personally. I would probably use something more soft:

> This client accepts an optional service API version parameter. By default the client library targets the latest service API version

Reply:
hello @lmazue
l think we want to tell SDK user that this client library is generated based on the latest api version, but a client instance can specify which api-version to communicate.  So

I change it to `The client library targets the latest service API version. A client instance accepts an optional service API version parameter to specify which API version service to communicate, by default, it is the latest API version.` 
What’s your opinion about this?

Reply:
updated to `The client library targets the latest service API version by default. A client instance accepts an optional service API version parameter to specify which API version service to communicate.` as discussed in email.